### ISSUE TITLE
test: use `just` to unify local test and CI test

### ISSUE THREAD

**Issue by MrCroxx** [foyer-rs/foyer#787]

**Comment by caicancai** [comment]
take, try vibe coding

**Comment by MrCroxx** [comment]
Oh, sorry. This is no longer needed after #1043 . 🥹

**Comment by MrCroxx** [comment]
No longer needed after #1043
